Tasks/Task2/query04.sql

```diff
@@ -0,0 +1,14 @@
+﻿--Query 4
+SELECT
+    pp.BusinessEntityID,
+    pp.LastName,
+    pp.FirstName
+FROM Person.Person AS pp
+WHERE 
+    pp.BusinessEntityID NOT IN (SELECT BusinessEntityID FROM Person.BusinessEntityAddress) 
```

Comment:
sub-queries would make it expensive
check exec plan to compare performance joins vs subqueries